src/newspaperOrderRule.ts

```diff
@@ -182,6 +186,7 @@ abstract class NewspaperHelper {
                         const edge = [methodName, depName];
                         graph.push(edge);
                     });
+                    graph.push([methodName, '--end-of-dependencies-marker--']);
```

Comment:
@lukaselmer : Please move `'--end-of-dependencies-marker--'` into a reused variable.